### ISSUE TITLE
API links in endpoints howto are broken

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4774]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#4774|
|Type|defect|
|Created|2010-12-14 19:01:05Z|
|Branch|https://github.com/twisted/twisted/tree/endpoints-doc-links-4774|

The markup for most API links appears to be incorrect, using a class of "api" instead of "API".  As a result, the generated documentation doesn't contain links.

Attachments:

* [twisted-ticket-4774.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/72f/72f98241eafcb4012285d1afdf46b12021f3cf10/d13fd2be365d5696c83a9569d30741b6ec0e05ed.patch) (7610 bytes) - added by tpratt on 2011-01-07 00:05:58Z - Does this do the trick?

<details><summary>Searchable metadata</summary>

```
trac-id__4774 4774
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__branches_endpoints_doc_links_4774 branches/endpoints-doc-links-4774
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation__easy documentation, easy
time__1292353265000000 1292353265000000
changetime__1294410986000000 1294410986000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [30426]) Branching to 'endpoints-doc-links-4774'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

(In [30427]) Apply patch fixing links

refs [#4774](https://github.com/twisted/twisted/issues/4774)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set owner to @exarkun<br>@exarkun set status to `assigned`|
|-|-|

Thanks!  Looks good to me.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set status to `closed`|
|-|-|

(In [30429]) Merge endpoints-doc-links-4774

Author: tpratt
Reviewer: exarkun
Fixes: [#4774](https://github.com/twisted/twisted/issues/4774)

Correct the Lore markup for the API references in the endpoints howto
so that links are actually generated.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
